src/actions/actions.js

```diff
@@ -0,0 +1,53 @@
+const databaseName = 'todoz'
+const databaseName_test = 'todoz_test'
+let connectionString
+
+if(process.env.NODE_ENV === 'development'){
+  connectionString = `postgres://${process.env.USER}@localhost:5432/${databaseName}`
+} else if (process.env.NODE_ENV === 'test'){
+  connectionString = `postgres://${process.env.USER}@localhost:5432/${databaseName_test}`
+}
+
+const pgp = require('pg-promise')();
+const db = pgp( connectionString );
+
```

Comment:
all these lines above should be in the db.js file, which in turn would export `db` then the file(s) in the actions folder would have something like `const db = require('../database/db)` in it and also export `db`